File: find_list.py
import requests
import json
import os

API_TOKEN = os.environ.get('CLICKUP_API_TOKEN', 'pk_87908883_PNJX99ZBG343SRG53Z7AKLEKLBOQCOQU')
HEADERS = {'Authorization': API_TOKEN, 'Content-Type': 'application/json'}
BASE_URL = 'https://api.clickup.com/api/v2'

res = requests.get(f"{BASE_URL}/team/9013412527/space", headers=HEADERS)
spaces = res.json().get('spaces', [])
for space in spaces:
    print(f"Space: {space['name']} ({space['id']})")
    if "Ative 360" in space['name']:
        res_folders = requests.get(f"{BASE_URL}/space/{space['id']}/folder", headers=HEADERS)
        folders = res_folders.json().get('folders', [])
        for folder in folders:
            print(f"  Folder: {folder['name']} ({folder['id']})")
            if "Apoio" in folder['name']:
                res_lists = requests.get(f"{BASE_URL}/folder/{folder['id']}/list", headers=HEADERS)
                lists = res_lists.json().get('lists', [])
                for lst in lists:
                    print(f"    List: {lst['name']} ({lst['id']})")

File: find_list.ts
import fetch from 'node-fetch';

const API_TOKEN = 'pk_87908883_PNJX99ZBG343SRG53Z7AKLEKLBOQCOQU';
const HEADERS = { 'Authorization': API_TOKEN, 'Content-Type': 'application/json' };
const BASE_URL = 'https://api.clickup.com/api/v2';

async function run() {
  const spaceId = '901313601110';
  const resFolders = await fetch(`${BASE_URL}/space/${spaceId}/folder`, { headers: HEADERS });
  const foldersData = await resFolders.json();
  const folders = foldersData.folders || [];
  for (const folder of folders) {
    console.log(`  Folder: ${folder.name} (${folder.id})`);
    const resLists = await fetch(`${BASE_URL}/folder/${folder.id}/list`, { headers: HEADERS });
    const listsData = await resLists.json();
    const lists = listsData.lists || [];
    for (const lst of lists) {
      console.log(`    List: ${lst.name} (${lst.id})`);
    }
  }
  
  const resLists = await fetch(`${BASE_URL}/space/${spaceId}/list`, { headers: HEADERS });
  const listsData = await resLists.json();
  const lists = listsData.lists || [];
  for (const lst of lists) {
    console.log(`  Folderless List: ${lst.name} (${lst.id})`);
  }
}

run();
